data/hai/hai.txt

```diff
@@ -11,28 +11,46 @@
 # You should have received a copy of the GNU General Public License along with
 # this program. If not, see <https://www.gnu.org/licenses/>.
 
-shipyard "Hai Basics"
-	"Aphid"
-	"Lightning Bug"
+# Imo Loo Mer Corp shipyards:
+shipyard "Imo Loo Mer Showroom"
 	"Centipede"
-	"Grasshopper"
+	"Geocoris"
 
-shipyard "Hai Intermediate"
-	"Aphid"
+shipyard "Imo Loo Mer Export"
+	"Scarab"
 	"Lightning Bug"
-	"Water Bug"
-	"Centipede"
-	"Grasshopper"
 
-shipyard "Hai Advanced"
+
+# Mon Ki o'Uden Splinter (unfettered) shipyards:
+shipyard "Mon Ki o'Uden Showroom"
 	"Aphid"
-	"Lightning Bug"
-	"Water Bug"
+	"Lightning Bug"	#Stolen
+	"Sea Scorpion"
 	"Shield Beetle"
-	"Centipede"
-	"Geocoris"
+	# Has "Solifuge", "Violin Spider" and "Roach"(TBD) added by event.
```

Comment:
Don't mention a ship that isn't in this PR or in the game already.
```suggestion
	# Has "Solifuge" and "Violin Spider" added by event.
```

Reply:
Actually, I don't think the Solifuge and Violin Spider actually are added to any shipyard at the moment? And, this PR doesn't change that, so this comment maybe just shouldn't be here?

Reply:
Mmm, they are *associated* with this shipyard though. Comment to that effect?

Reply:
Perhaps. Or, we could just add it with an event? I'm pretty sure the Solifuge already requires a license so the player wouldn't be able to buy it, just look at it if they go wherever this shipyard is. Not sure about the fighter, though.

Reply:
If it gets added to the shipyard itself by event it will be the purview of the UHai campaign, not this PR.

Reply:
Yep.
It'll mostly be produced on Wanderer worlds except if you do the Uhai campaign
But thats a worry for part II or III of the campaign